diff --git a/components/policy/core/common/BUILD.gn b/components/policy/core/common/BUILD.gn
index a09be30459452..cdc9cf1d54058 100644
--- a/components/policy/core/common/BUILD.gn
+++ b/components/policy/core/common/BUILD.gn
@@ -497,7 +497,10 @@ static_library("test_support") {
 
 source_set("common_constants") {
   configs += [ "//components/policy:component_implementation" ]
-  deps = [ "//base" ]
+  deps = [
+    "//base",
+    "//teleport:teleport_policy_buildflags",
+  ]
   sources = [
     "../../policy_export.h",
     "cloud/cloud_policy_constants.cc",
