diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 1a692fe440945..5a81ef8e76bdc 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -71,7 +71,10 @@ source_set("channel_info") {
     "channel_info.h",
   ]
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
-  deps = [ "//build:branding_buildflags" ]
+  deps = [
+    "//build:branding_buildflags",
+    "//teleport",
+  ]
   public_deps = [
     "//base",
     "//components/version_info",
@@ -258,6 +261,7 @@ static_library("common_lib") {
   ]
 
   deps = [
+    "//teleport",
     "//components/cdm/common:buildflags",
     "//components/crash/core/app",
     "//components/google/core/common",
