diff --git a/net/base/test_proxy_delegate.cc b/net/base/test_proxy_delegate.cc
index 319e45768ce51..210d9e659cce7 100644
--- a/net/base/test_proxy_delegate.cc
+++ b/net/base/test_proxy_delegate.cc
@@ -161,6 +161,17 @@ std::string TestProxyDelegate::GetExtraHeaderValue(
   return ProxyServerToProxyUri(proxy_server);
 }
 
+void TestProxyDelegate::OnBeforeForwardProxyRequest(
+    const ProxyChain& proxy_chain,
+    HttpRequestHeaders* extra_headers) {
+  on_before_forward_proxy_request_call_count_++;
+
+  if (extra_header_name_) {
+    extra_headers->SetHeader(*extra_header_name_,
+                             GetExtraHeaderValue(proxy_chain.First()));
+  }
+}
+
 base::expected<HttpRequestHeaders, Error>
 TestProxyDelegate::OnBeforeTunnelRequest(
     const ProxyChain& proxy_chain,
