diff --git a/chrome/test/BUILD.gn b/chrome/test/BUILD.gn
index a7bd79373d003..4fd5077b4fe23 100644
--- a/chrome/test/BUILD.gn
+++ b/chrome/test/BUILD.gn
@@ -5087,6 +5087,7 @@ test("unit_tests") {
     "../browser/chrome_content_browser_client_parts_unittest.cc",
     "../browser/chrome_content_browser_client_unittest.cc",
     "../browser/command_updater_impl_unittest.cc",
+    "//teleport/browser/enterprise/teleport_tunnel_service_unittest.cc",
     "../browser/file_select_helper_unittest.cc",
     "../browser/font_family_cache_unittest.cc",
     "../browser/font_pref_change_notifier_unittest.cc",
@@ -5195,6 +5196,7 @@ test("unit_tests") {
     ":test_support",
     ":test_support_unit",
     "//base:base_stack_sampling_profiler_test_util",
+    "//teleport",
     "//base/test:test_support",
     "//chrome:resources",
     "//chrome:strings",
