diff --git a/chrome/browser/resources/settings/about_page/about_page_browser_proxy.ts b/chrome/browser/resources/settings/about_page/about_page_browser_proxy.ts
index 10bea4664af53..18c86879aed3f 100644
--- a/chrome/browser/resources/settings/about_page/about_page_browser_proxy.ts
+++ b/chrome/browser/resources/settings/about_page/about_page_browser_proxy.ts
@@ -59,14 +59,11 @@ export interface AboutPageBrowserProxy {
   /** Opens the help page. */
   openHelpPage(): void;
 
-  // <if expr="_google_chrome">
   /**
    * Opens the feedback dialog.
    */
   openFeedbackDialog(): void;
 
-  // </if>
-
   // <if expr="_google_chrome and is_macosx">
   /**
    * Triggers setting up auto-updates for all users.
@@ -94,11 +91,9 @@ export class AboutPageBrowserProxyImpl implements AboutPageBrowserProxy {
     chrome.send('openHelpPage');
   }
 
-  // <if expr="_google_chrome">
   openFeedbackDialog() {
     chrome.send('openFeedbackDialog');
   }
-  // </if>
 
   static getInstance(): AboutPageBrowserProxy {
     return instance || (instance = new AboutPageBrowserProxyImpl());
