diff --git a/third_party/lit/v3_0/BUILD.gn b/third_party/lit/v3_0/BUILD.gn
index e58a9be1616fd..0a6f095b8bd2e 100644
--- a/third_party/lit/v3_0/BUILD.gn
+++ b/third_party/lit/v3_0/BUILD.gn
@@ -171,6 +171,12 @@ ts_library("build_ts") {
     "//ui/webui/resources/cr_components/searchbox:build_ts",
     "//ui/webui/resources/cr_components/theme_color_picker:build_ts",
     "//ui/webui/resources/cr_elements:build_ts",
+
+    # teleport overlay: chrome://enroll Lit page (build_webui).
+    "//teleport/browser/resources/enroll:build_ts",
+
+    # teleport overlay: chrome://tunnel Lit page (build_webui).
+    "//teleport/browser/resources/tunnel:build_ts",
   ]
   tsconfig_base = "tsconfig_base.json"
   composite = true
