diff --git a/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc b/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc
index b97220b8d91e6..cf6f9e3034de0 100644
--- a/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc
+++ b/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc
@@ -46,6 +46,8 @@ std::string_view GetStepHistogramSuffix(
       return ".SearchEngineChoice";
     case ProfileManagementFlowController::Step::kFinishFlow:
       return ".FinishFlow";
+    case ProfileManagementFlowController::Step::kTeleportEnrollment:
+      return ".TeleportEnrollment";
   }
 }
 // LINT.ThenChange(//tools/metrics/histograms/metadata/profile/histograms.xml:StepName)
