diff --git a/chrome/browser/ui/views/profiles/profile_picker_view.cc b/chrome/browser/ui/views/profiles/profile_picker_view.cc
index a6a323eedcd7e..6b7d990b0a788 100644
--- a/chrome/browser/ui/views/profiles/profile_picker_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_picker_view.cc
@@ -227,6 +227,17 @@ void ProfilePicker::SwitchToReauth(
   }
 }
 
+// static
+void ProfilePicker::SwitchToEnrollment(
+    Profile* profile,
+    base::OnceCallback<void(bool)> switch_finished_callback) {
+  if (g_profile_picker_view) {
+    g_profile_picker_view->SwitchToEnrollment(
+        profile,
+        StepSwitchFinishedCallback(std::move(switch_finished_callback)));
+  }
+}
+
 // static
 void ProfilePicker::SwitchToSignedOutPostIdentityFlow(
     std::optional<SkColor> profile_color) {
@@ -754,6 +765,13 @@ void ProfilePickerView::SwitchToReauth(
       std::move(on_error_callback));
 }
 
+void ProfilePickerView::SwitchToEnrollment(
+    Profile* profile,
+    StepSwitchFinishedCallback switch_finished_callback) {
+  GetProfilePickerFlowController()->SwitchToEnrollment(
+      profile, std::move(switch_finished_callback));
+}
+
 void ProfilePickerView::WindowClosing() {
   // If a profile is locked, it might have been loaded and it's first browser
   // will never be created, we need to remove it's equivalent
