diff --git a/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc b/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc
index d2c02af720b3c..ea772ad8888f8 100644
--- a/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc
+++ b/chrome/browser/ui/views/profiles/profile_management_flow_controller.cc
@@ -53,6 +53,8 @@ std::string_view GetStepHistogramSuffix(
       return ".FinishOrContinue";
     case ProfileManagementFlowController::Step::kDeviceSignalsDisclaimer:
       return ".DeviceSignalsDisclaimer";
+    case ProfileManagementFlowController::Step::kTeleportEnrollment:
+      return ".TeleportEnrollment";
   }
 }
 // LINT.ThenChange(//tools/metrics/histograms/metadata/profile/histograms.xml:StepName)
