diff --git a/chrome/app/resources/generated_resources_zh-CN.xtb b/chrome/app/resources/generated_resources_zh-CN.xtb
index dac84ad637c50..5fd75069298b0 100644
--- a/chrome/app/resources/generated_resources_zh-CN.xtb
+++ b/chrome/app/resources/generated_resources_zh-CN.xtb
@@ -11964,4 +11964,6 @@
 <translation id="998347228911661591">同步标签页分组</translation>
 <translation id="998747458861718449">检查(&amp;N)</translation>
 <translation id="998852291477823323">配置的政策不允许执行此操作。</translation>
+<translation id="4486075783877917896">登录后由你的组织统一管理此浏览器</translation>
+<translation id="5964625613525083954">由 <ph name="ORGANIZATION" /> 管理</translation>
 </translationbundle>
\ No newline at end of file
