diff --git a/components/policy/core/browser/BUILD.gn b/components/policy/core/browser/BUILD.gn
index 2f4c0e097fa51..c3f2de28837bd 100644
--- a/components/policy/core/browser/BUILD.gn
+++ b/components/policy/core/browser/BUILD.gn
@@ -104,6 +104,7 @@ source_set("internal") {
     "//net",
     "//services/network/public/cpp",
     "//third_party/re2",
+    "//teleport:teleport_deployment_config",
     "//ui/base",
     "//url",
   ]
