diff --git a/components/policy/core/browser/policy_conversions.h b/components/policy/core/browser/policy_conversions.h
index f67c0b96ed172..fbd49fe42c78d 100644
--- a/components/policy/core/browser/policy_conversions.h
+++ b/components/policy/core/browser/policy_conversions.h
@@ -47,7 +47,7 @@ inline constexpr char kNameKey[] = "name";
 inline constexpr char kPoliciesKey[] = "policies";
 inline constexpr char kPolicyNamesKey[] = "policyNames";
 inline constexpr char kChromePoliciesId[] = "chrome";
-inline constexpr char kChromePoliciesName[] = "Chrome Policies";
+inline constexpr char kChromePoliciesName[] = "Teleport Policies";
 inline constexpr char kExtensionInstallPoliciesId[] = "extensionInstall";
 inline constexpr char kExtensionInstallPoliciesName[] =
     "Extension Install Policies";
